VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -46,10 +49,21 @@ def __init__(self,patching, hutil, logger,para_parser):
             errMsg = 'Failed to serialize customSettings with error: %s, stack trace: %s' % (str(e), traceback.format_exc())
             self.logger.log(errMsg, True, 'Error')
             self.isOnlyOSDiskBackupEnabled = False
+        self.command="sudo lsscsi"
+        self.dict1={}
+        self.lun1=[]
+        self.p=(os.popen(self.command).read())
```

Comment:
should this be wrapped in a try-catch block in case we get a failure? Please check if we should continue to follow older logic in case of exception.